### ISSUE TITLE
Making the App Responsive and Look Awesome on Mobile Devices

### ISSUE THREAD

**Issue by kenmazaika** [knoll3/Splurty#1]
On mobile things don't look as good as they could be.

One thing you could look into is to add the "mobile meta tag" to your app, so the responsive effect of Bootstrap actually shows up on Mobile.

Let me know in the comments once you figured that step out :)

**Comment by knoll3** [comment]
Thanks for the advice. I looked it up and I added the following code to the application.html.erb: 

`<meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1.0" />`

This allows Bootstrap's responsive effect to show up on Mobile. Looks a lot better!

Then I was having a problem with some phones allowing horizontal scrolling. Strictly vertical scrolling is much cleaner so I added the following code to master.scss: 

`body {
    overflow-x: hidden;
    width: 100%;
}
`
This fixed that problem so now it responds even better.

Thanks for the help. Now my app looks pretty slick on mobile!
